### ISSUE TITLE
Pinboard.in Bookmarklet Broken

### ISSUE THREAD

**Issue by chasingtheflow** [philc/vimium#1637]
The below javascript bookmarklet stopped working within Vimium recently, I'm not sure specifically which version:

```javascript
javascript:q=location.href;if(document.getSelection){d=document.getSelection();}else{d='';};p=document.title;void(open('https://pinboard.in/add?showtags=yes&url='+encodeURIComponent(q)+'&description='+encodeURIComponent(d)+'&title='+encodeURIComponent(p),'Pinboard','toolbar=no,scrollbars=yes,width=750,height=700'));
```

I do not know if it was a change in Vimium or a change in Chrome which caused this to break ... but I'd be grateful for any tips on how I can get this working again. For example, I have this saved as a bookmark so that I can launch it from Vimium `b pin ⏎` would launch the bookmarklet on the current page.

Thanks,

**Comment by smblott-github** [comment]
Thanks, @chasingtheflow.  See also #1611.

**Comment by mrmr1993** [comment]
This can probably be closed now the fix for #1611 has landed.
